packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -1853,6 +1895,12 @@ it('should disable row and handle selection accordingly', async () => {
   expect(headerCheckbox.indeterminate).toBe(true);
   expect(headerCheckbox.checked).toBe(false);
 
+  // Deselect all
```

Comment:
This code must be moved back to its original position

Reply:
**[DONE]**